cmd/kpod/push.go

```diff
@@ -76,15 +71,13 @@ func pushCmd(c *cli.Context) error {
 	destName := c.Args().Get(1)
 
 	signaturePolicy := c.String("signature-policy")
-	compress := archive.Uncompressed
-	if !c.Bool("disable-compression") {
-		compress = archive.Gzip
-	}
 	registryCredsString := c.String("creds")
 	certPath := c.String("cert-dir")
 	skipVerify := !c.BoolT("tls-verify")
 	removeSignatures := c.Bool("remove-signatures")
 	signBy := c.String("sign-by")
+	// Leave this here in case we want to add a compression option in the future
+	compress := archive.Uncompressed
```

Comment:
(I don’t really see that leaving the variable around is simplifying anything—sure, someone decided to re-add the CLI flag, it wouldn’t be necessary to reintroduce the variable, but in the mean time everyone will pay the price by seeing this single-use variable and having to read the comment.

But it does work just fine, and I guess separating the policy decisions from the implementation code can be considered a good design…)